handbook/engineering/security/goals.md

```diff
@@ -2,38 +2,51 @@
 
 ## Goals
 
-These goals represent our targeted work for 70% of our time. The remaining 30% is reserved for items that arise such as security reports. Our current work is documented in our [tracking issue](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+label%3Atracking+label%3Ateam%2Fsecurity+is%3Aopen).
+These goals represent our targeted work for 70% of our time. The remaining 30% is reserved for items that arise such as security reports. Our current work is documented in a [private GitHub project](https://github.com/orgs/sourcegraph/projects/130).
 
-### Visibility into Sourcegraph Cloud's attack surface
 
-**Problem:** The Cloud team is working toward [private code on Sourcegraph Cloud](../cloud/index.md#private-code-on-sourcegraph-cloud). Before we can have [confidence in our security model](#confidence-in-our-security-model), we first need to have visibility into current risks, issues, and threats, so we can proactively plan what work needs to be done (i.e., we don't want to be relying on or waiting for security researchers or customer to report vulnerabilities to us).
 
-**Milestones:**
+### Enable Sourcegraph Cloud
```

Comment:
Would "Secure Sourcegraph Cloud" be more specific?

Reply:
I'm going to try breaking this into two goals (secure for private code launch, secure for SMBs), since that gives us a much nearer term concrete "done" state, and is much more specific about what work we're doing and why.